src/components/table/_mixins.scss

```diff
@@ -6,7 +6,7 @@
 
 @mixin euiTableCellCheckbox {
   @include euiTableCell;
-  border-top: none;
+  // border-top: none;
```

Comment:
Was this intentional? I assume so, just double checking

Reply:
Yeah, that line should actually be deleted entirely.